### ISSUE TITLE
Ruby 2.7 deprecation warning

### ISSUE THREAD

**Issue by askrynnikov** [collectiveidea/awesome_nested_set#425]
Relevant stack/libraries:
awesome_nested_set - 3.2.0
Rails - 6.0.3.1
Ruby - 2.7.0

Loading my application since upgrading to Ruby 2.7 yields the following warning message:
gems/awesome_nested_set-3.2.0/lib/awesome_nested_set/awesome_nested_set.rb:105: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call

**Comment by amorozdna** [comment]
Yep, the same for me, in two places though.
```
.rvm/gems/ruby-2.7.1/bundler/gems/awesome_nested_set-32f25e5c15e3/lib/awesome_nested_set/awesome_nested_set.rb:105: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
.rvm/gems/ruby-2.7.1/gems/activerecord-6.0.3.2/lib/active_record/associations.rb:1657: warning: The called method `belongs_to' is defined here

.rvm/gems/ruby-2.7.1/bundler/gems/awesome_nested_set-32f25e5c15e3/lib/awesome_nested_set/awesome_nested_set.rb:90: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
.rvm/gems/ruby-2.7.1/gems/activerecord-6.0.3.2/lib/active_record/associations.rb:1370: warning: The called method `has_many' is defined here
```
That's for the master branch version of awesome_nested_set.

**Comment by benwyrosdick** [comment]
same

```
/Users/benjaminwyrosdick/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/awesome_nested_set-3.2.1/lib/awesome_nested_set/awesome_nested_set.rb:105: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/Users/benjaminwyrosdick/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/activerecord-6.0.3.2/lib/active_record/associations.rb:1657: warning: The called method `belongs_to' is defined here
/Users/benjaminwyrosdick/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/awesome_nested_set-3.2.1/lib/awesome_nested_set/awesome_nested_set.rb:90: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/Users/benjaminwyrosdick/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/activerecord-6.0.3.2/lib/active_record/associations.rb:1370: warning: The called method `has_many' is defined here
```

**Comment by parndt** [comment]
Fixed by #433
